### PR TITLE
Add save cover button functionality

### PR DESCRIPTION
**Pull request by CaliHam**
- When a user clicks the “Save Cover” button, the current cover is added to the savedCovers array
- If a user clicks the “Save Cover” more than once on a single cover, it will only be saved once (no duplicates)

### CODE REVIEW

**Comment by CaliHam** [comment]
Last update changes the Make Book Button functionality issue when trying to create multiple different books; the second book was not showing on the home view.

File: src/main.js [hunk 1]
```diff
@@ -69,18 +69,18 @@ function getRandomDesc() {
 }
 
 function getRandomCover() {
-  var randomCover = createCover(
+  currentCover = createCover(
```
Comment (mbenfowler) [inline src/main.js:72]: yep!

File: src/main.js [hunk 2]
```diff
@@ -98,15 +98,24 @@ function makeCoverButton() {
...
+  savedView.classList.add('hidden');
 
   formView.classList.remove('hidden')
   homeButton.classList.remove('hidden');
 }
 
+function saveCover() {
+  if (!savedCovers.includes(currentCover)) {
+    savedCovers.push(currentCover);
+    console.log(savedCovers);
```
Comment (mbenfowler) [inline src/main.js]: please remove all console logs! These should never make it into production code 😄

**Review by mbenfowler — CHANGES_REQUESTED** [review]
Just one change needed!  I also noticed that our database isn't maintaining previously saved covers other than that one hardcoded - and it looks like this is considered in iteration4:

`Hint: How will you update the data model to achieve this?`

So yah, might need your help looking into how we can persist our data tomorrow 😅 

EDIT** - So it looks like our saved covers are persisting afterall, as long as the page isn't refreshed.

File: src/main.js (hunk 2 above)
Comment (CaliHam) [reply src/main.js]: Oops! Done!

**Review by mbenfowler — APPROVED** [review]
Woooooo

**Pull request merged**